app/src/main/java/hm/orz/chaos114/android/slideviewer/ui/SelectOcrLanguageActivity.java

```diff
@@ -83,7 +83,7 @@ protected void onCreate(Bundle savedInstanceState) {
         binding.languageList.setAdapter(adapter);
     }
 
-    private class Adapter extends BaseAdapter {
+    private class Adapter extends BaseAdapter { 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/noboru-i/SlideViewer/blob/44179ee7e2abf2541e63aa2912d8fb40b96a7a57/app/src/main/java/hm/orz/chaos114/android/slideviewer/ui/SelectOcrLanguageActivity.java#L86"></span>Class Adapter should be declared as final.</td>
    </tr>
  </tbody>
</table>